api/api.py

```diff
@@ -9,99 +9,204 @@
 
 from . import core
 from . import jobs
-from . import users
-from . import projects
-from . import sessions
-from . import acquisitions
-from . import collections
+from handlers import listhandler
+from handlers import userhandler
+from handlers import grouphandler
+from handlers import containerhandler
+from handlers import collectionshandler
 
+#regexes used in routing table:
+routing_regexes = {
+    'group_id_re': '[0-9a-z][0-9a-z.@_-]{0,30}[0-9a-z]',
+    'cid_re': '[0-9a-f]{24}',
+    'site_re': '[0-9a-z]{0,24}',
+    'user_id_re': '[0-9a-z.@_-]*',
+    'cont_name_re': 'projects|sessions|acquisitions|collections',
+    'tag_re': '[^/]{3,24}',
+    'filename_re': '[^/]{3,60}',
+    'note_id_re': '[0-9a-f]{24}'
+}
+
+def _format(route):
+    return route.format(**routing_regexes)
 
 routes = [
-    webapp2.Route(r'/api',                                          core.Core),
+    webapp2.Route(
+        r'/api',
+        core.Core
+    ),
     webapp2_extras.routes.PathPrefixRoute(r'/api', [
-        webapp2.Route(r'/download',                                 core.Core, handler_method='download', methods=['GET', 'POST'], name='download'),
-        webapp2.Route(r'/upload',                                   core.Core, handler_method='upload', methods=['POST']),
-        webapp2.Route(r'/reaper',                                   core.Core, handler_method='reaper', methods=['POST']),
-        webapp2.Route(r'/sites',                                    core.Core, handler_method='sites', methods=['GET']),
-        webapp2.Route(r'/search',                                   core.Core, handler_method='search', methods=['GET', 'POST']),
-    ]),
-    webapp2_extras.routes.PathPrefixRoute(r'/api/schema', [
-        webapp2.Route(r'/group',                                    users.Group, handler_method='schema', methods=['GET']),
-        webapp2.Route(r'/user',                                     users.User, handler_method='schema', methods=['GET']),
+        webapp2.Route(
```

Comment:
Suggestion to format these routes as before, despite the long line length. It is less readable now.